recipes/lazy_loader/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "lazy_loader" %}
+{% set version = "0.1rc2" %}
```

Comment:
This should probably target a pre-release channel: https://conda-forge.org/docs/maintainer/knowledge_base.html#pre-release-builds

Reply:
Also see here: https://github.com/conda-forge/staged-recipes/pull/18695#issue-1207235256

Reply:
Oops!

Reply:
So, what do you suggest now? Can this PR be merged? (Given, the other PR you shared)

Reply:
> This should probably target a pre-release channel: https://conda-forge.org/docs/maintainer/knowledge_base.html#pre-release-builds

I will update the recipe with pre-release build. Thank you.

Reply:
@BastianZim Done.

A question: So, for any downstream recipe, will the following be necessary to use `lazy_loader` from the `pre-release` channel? Also, how does this change/impact the downstream recipe, if later on the `lazy_loader` recipe points to a non `rc` PyPI version? 

```yml
# file: recipe/conda_build_config.yaml
channel_sources:
  - conda-forge/label/lazy_loader_rc,conda-forge
```

Reply:
Yes, it will be, which is why most wait until a non rc build is available.

If the recipe is non rc the normal selection will apply, meaning that mamba will select the highest number from both channels or focus on whichever channel you have as your first one as determined by your settings: https://mamba.readthedocs.io/en/latest/advanced_usage/package_resolution.html

Reply:
Let me know if you want me to merge the rc version of if you want to wait.

Reply:
@BastianZim Yes, please merge the rc version.